boot/CMakeLists.txt

```diff
@@ -59,106 +61,112 @@ set(ISO_VOLNAME      "ReactOS")             # For both the Volume ID and the Vol
+    set_property(TARGET ${_target} APPEND PROPERTY FILE_LIST
+        "${_rootdir}/${_username}/Application Data=${CMAKE_CURRENT_BINARY_DIR}/empty"
+        "${_rootdir}/${_username}/Application Data/Microsoft/Internet Explorer/Quick Launch=${CMAKE_CURRENT_BINARY_DIR}/empty"
+        "${_rootdir}/${_username}/Cookies=${CMAKE_CURRENT_BINARY_DIR}/empty"
+        "${_rootdir}/${_username}/Desktop=${CMAKE_CURRENT_BINARY_DIR}/empty"
+        "${_rootdir}/${_username}/Favorites=${CMAKE_CURRENT_BINARY_DIR}/empty"
+        "${_rootdir}/${_username}/Local Settings/Application Data=${CMAKE_CURRENT_BINARY_DIR}/empty"
+        "${_rootdir}/${_username}/Local Settings/History=${CMAKE_CURRENT_BINARY_DIR}/empty"
+        "${_rootdir}/${_username}/Local Settings/Temporary Internet Files=${CMAKE_CURRENT_BINARY_DIR}/empty"
+        "${_rootdir}/${_username}/My Music=${CMAKE_CURRENT_BINARY_DIR}/empty"
+        "${_rootdir}/${_username}/My Pictures=${CMAKE_CURRENT_BINARY_DIR}/empty"
+        "${_rootdir}/${_username}/My Videos=${CMAKE_CURRENT_BINARY_DIR}/empty"
+        "${_rootdir}/${_username}/NetHood=${CMAKE_CURRENT_BINARY_DIR}/empty"
+        "${_rootdir}/${_username}/PrintHood=${CMAKE_CURRENT_BINARY_DIR}/empty"
+        "${_rootdir}/${_username}/Recent=${CMAKE_CURRENT_BINARY_DIR}/empty"
+        "${_rootdir}/${_username}/SendTo=${CMAKE_CURRENT_BINARY_DIR}/empty"
+        "${_rootdir}/${_username}/Start Menu/Programs=${CMAKE_CURRENT_BINARY_DIR}/empty"
+        "${_rootdir}/${_username}/Start Menu/Programs/Administrative Tools=${CMAKE_CURRENT_BINARY_DIR}/empty"
+        "${_rootdir}/${_username}/Start Menu/Programs/StartUp=${CMAKE_CURRENT_BINARY_DIR}/empty"
+        "${_rootdir}/${_username}/Templates=${CMAKE_CURRENT_BINARY_DIR}/empty")
 endfunction()
 
 
 ## BootCD
-# Create the file list
-file(WRITE ${CMAKE_CURRENT_BINARY_DIR}/bootcd.cmake.lst "")
-file(WRITE ${CMAKE_CURRENT_BINARY_DIR}/bootcd.cmake.lst "${CMAKE_CURRENT_BINARY_DIR}/empty\n")
-
-add_custom_target(bootcd
+file(GENERATE
+    OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/bootcd.$<CONFIG>.lst
+    CONTENT $<GENEX_EVAL:$<JOIN:$<TARGET_PROPERTY:bootcd,FILE_LIST>,\n>>\n)
+add_custom_command(
+    OUTPUT ${REACTOS_BINARY_DIR}/bootcd.iso
     COMMAND native-mkisofs -quiet -o ${REACTOS_BINARY_DIR}/bootcd.iso -iso-level 4
         -publisher ${ISO_MANUFACTURER} -preparer ${ISO_MANUFACTURER} -volid ${ISO_VOLNAME} -volset ${ISO_VOLNAME}
         -eltorito-boot loader/isoboot.bin -no-emul-boot -boot-load-size 4 -eltorito-alt-boot -eltorito-platform efi -eltorito-boot loader/efisys.bin -no-emul-boot -hide boot.catalog
         -sort ${CMAKE_CURRENT_BINARY_DIR}/bootfiles.sort
         -no-cache-inodes -graft-points -path-list ${CMAKE_CURRENT_BINARY_DIR}/bootcd.$<CONFIG>.lst
-    COMMAND native-isohybrid -b ${_isombr_file} -t 0x96 ${REACTOS_BINARY_DIR}/bootcd.iso
-    DEPENDS isombr native-isohybrid native-mkisofs
+    DEPENDS isombr native-isohybrid native-mkisofs $<GENEX_EVAL:$<TARGET_PROPERTY:bootcd,ISO_DEPENDENCIES>>
```

Comment:
It seems that the correct native-isohybrid command call just disappeared ... :(
 (while it remains for the livecd and the others, as expected)

Reply:
Right. Copy-paste fail here.